### ISSUE TITLE
After upgrade to 3.0.4 app doesn't work

### ISSUE THREAD

**Issue by madmak2005** [julien-nc/cospend-nc#319]
I can add new Project and nothing more is possible. I uninstaled it and installed again but didn't help
![image](https://github.com/user-attachments/assets/f2fbd91f-401f-46e5-bc52-9c426a1b6b20)
I have Nexcloud version 30.0.4

**Comment by julien-nc** [comment]
Can you confirm it is fixed in v3.0.5?

It was a weird issue with the "vite" build.

**Comment by tpokorra** [comment]
I still get an error message, when updating the app:

```
php occ app:update --all -n --no-ansi

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
cospend new version available: 3.0.5
Error: Database error when running migration 030000Date20240910234723 for app cospend
An exception occurred while executing a query: SQLSTATE[23502]: Not null violation: 7 ERROR:  column "user_id" of relation "oc_cospend_projects" contains null values
cospend couldn't be updated
```
